Material.Ripple/RippleEffect.cs

```diff
@@ -110,6 +120,25 @@ private void RemoveLastRipple() {
             _last = null;
         }
 
+        public void RaiseRipple(double nX = 0.5, double nY = 0.5) {
+            var c = _container;
+
+            if (c is null || nX < 0 || nX > 1 || nY < 0 || nY > 1)
+                return;
```

Comment:
This should be exception (in the method with 0-1 range values), not just swallowing and doing nothing.

Reply:
added line `throw new ArgumentOutOfRangeException();`